<launch>
	<master auto="start"/>

	<node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
		<param name="controller_frequency" value="2.0" />
		<rosparam file="$(find fembots)/config/simulation/costmap_common_params.yaml" command="load" ns="global_costmap" />
		<rosparam file="$(find fembots)/config/simulation/costmap_common_params.yaml" command="load" ns="local_costmap" />
		<rosparam file="$(find fembots)/config/simulation/local_costmap_params.yaml" command="load" />
		<rosparam file="$(find fembots)/config/simulation/global_costmap_params.yaml" command="load" />
		<rosparam file="$(find fembots)/config/simulation/base_local_planner_params.yaml" command="load" />
	</node>

</launch>
